aws-greengrass-testing-api/src/main/java/com/aws/greengrass/testing/api/device/local/LocalDevice.java

```diff
@@ -59,7 +59,13 @@ public String type() {
 
     @Override
     public byte[] execute(CommandInput input) throws CommandExecutionException {
-        final ProcessBuilder builder = new ProcessBuilder().command(input.line());
+        final ProcessBuilder builder;
```

Comment:
```suggestion
        final ProcessBuilder builder = new ProcessBuilder().command(input.line());
        
        if (input.line().contains(WINDOWS_RUN_COMMAND)) {
             builder.command(WINDOW_RUN_COMMAND + " /c");
        }
```

Reply:
> 

This will not work. We need to pass "cmd.exe" and "/c" separately to ProcessBuilder. Otherwise we will hit the issue: ``` Cannot run program "cmd.exe /c": CreateProcess error=2, The system cannot find the file specified```

Reply:
Well then 

```
     builder.command(WINDOW_RUN_COMMAND).add("/c")
```

Reply:
+1

Reply:
Done